include/deal.II/base/config.h.in

```diff
@@ -116,7 +116,21 @@
  */
 
 #cmakedefine DEAL_II_WORDS_BIGENDIAN
-#define DEAL_II_COMPILER_VECTORIZATION_LEVEL @DEAL_II_COMPILER_VECTORIZATION_LEVEL@
+#define DEAL_II_HIGHEST_N_VECTORIZATION_BITS @DEAL_II_HIGHEST_N_VECTORIZATION_BITS@
```

Comment:
Would you mind adding a comment here documenting what the macro is supposed to be represent (or where else to find documentation)?